vulnerabilities/api_v2.py

```diff
@@ -41,11 +40,24 @@ class Meta:
         fields = ["url", "reference_type", "reference_id"]
 
 
+class VulnerabilitySeverityV2Serializer(serializers.ModelSerializer):
+    class Meta:
+        model = VulnerabilitySeverity
+        fields = ["url", "value", "scoring_system", "scoring_elements", "published_at"]
+
+    def to_representation(self, instance):
```

Comment:
Why we need `to_representation` method here? if published_at is None, let it be sent as None

Reply:
Added this just to preserve the response structure we're already using in APIv2 https://github.com/aboutcode-org/vulnerablecode/blob/8a68c97dfa369ad048de3ece14cc1b3cf40591cc/vulnerabilities/api_v2.py#L16C33-L16C64.

IMO yes we should simpy send `None` when we don't have published date.